src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -89,14 +89,18 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
   const radioStations = useSelector(radioStationsSelector);
   const parentVenue = useSelector(parentVenueSelector);
 
-  const venueParentId = venue?.parentId;
-  const venueParentQuery = useMemo<ReduxFirestoreQuerySetting>(
-    () => ({
-      collection: "venues",
-      doc: venueParentId,
-      storeAs: "parentVenue",
-    }),
-    [venueParentId]
+  const venueParentQuery = useMemo<SparkleRFDocQuery[]>(
```

Comment:
We don't need to useMemo on these queries, can we clean it up.

Reply:
[OOS, Future Improvement] Let's clean up this pattern in a new PR/as we come across it.